frontend/apps/picasso/defi/polkadot/hooks/useExistentialDeposit.ts

```diff
@@ -31,9 +32,21 @@ export const useExistentialDeposit = () => {
   useEffect(() => {
     switch (from) {
       case "karura":
-        if (["kusd", "ausd"].includes(tokenId)) {
-          updateExistentialDeposit(new BigNumber(1));
+        // @see https://wiki.acala.network/get-started/get-started/karura-account
+        const karuraEdMap: {
+          [key in string] : BigNumber
+        } = {
+          "kusd": new BigNumber(0.01),
+          "ausd": new BigNumber(0.01),
+          "kar": new BigNumber(0.1),
+          "ksm": new BigNumber(0.0001),
+        };
+        const ed = tokenId in karuraEdMap ? karuraEdMap[tokenId] : new BigNumber(1);
```

Comment:
```suggestion
        const existentialDeposit = karuraEdMap?.[tokenId] || new BigNumber(1);
```

Reply:
This wouldn't work since `tokenId` has more subset of values, causes TS to fail. the `tokenId in karuraEdMap` is for typescript to approve only allowed keys are being passed.